package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Should be part of web-jehlomat, I guess.

Reply:
?

Reply:
Just remove the package.json file from the root of the project.